lib/resty/upstream/healthcheck.lua

```diff
@@ -213,65 +217,52 @@ local function check_peer(ctx, id, peer, is_backup)
         errlog("failed to create stream socket: ", err)
         return
     end
-
+    
     sock:settimeout(ctx.timeout)
 
     if peer.host then
-        -- print("peer port: ", peer.port)
```

Comment:
Please do not remove such commented debugging statements since we may want to re-enable them (temporarily) for future debugging.